src/Navigation.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+export default function Navigation() {
+  const [toggle, setToggle] = useState(true)
```

Comment:
What's the purpose of this state? As far as I can tell, this doesn't do anything besides triggering the logging on click? Maybe you forgot to delete it?